src/yang/iana-bgp-types.yang

```diff
@@ -623,4 +623,83 @@ module iana-bgp-types {
        option 1: 4-byte number
        option 2: IP address";
   }
+
+  typedef graceful-restart-time-type {
+    type uint16 {
+      range "0..4096";
+    }
+    units "seconds";
+    description
+      "This is the estimated time (in seconds) it will take for the
+       BGP session to be re-established after a restart.  This can be
+       used to speed up routing convergence by its peer in case that
+       the BGP speaker does not come back after a restart.";
+    reference
+       "RFC 4724: Graceful Restart Mechanism for BGP, Section 3.";
+  }
+
+  typedef graceful-restart-flags {
+    type bits {
+      bit restart {
+        position 0;
+        description
+          "The most significant bit is defined as the Restart
+           State (R) bit, which can be used to avoid possible
+           deadlock caused by waiting for the End-of-RIB marker
+           when multiple BGP speakers peering with each other
+           restart.  When set (value 1), this bit indicates
+           that the BGP speaker has restarted, and its peer
+           MUST NOT wait for the End-of-RIB marker from the
+           speaker before advertising routing information to
+           the speaker.";
+        reference
+          "RFC 4724: Graceful Restart Mechanism for BGP,
+           Section 3.";
+      }
+      bit notification {
+        position 1;
+        description
+          "The second most significant bit is defined in [RFC 8538]
+           as the Graceful Notification ('N') bit.  It is used to
+           indicate Graceful Restart support for BGP NOTIFICATION
+           messages.  A BGP speaker indicates support for the
+           procedures in this document by advertising a Graceful
+           Restart Capability with its 'N' bit set (value 1).";
```

Comment:
Is 1). on a new line by itself and not properly indented, or is the rendering in the browser that makes me think that.?

Reply:
Looks fine in my browser and in the file.